cm15/api_data.json

```diff
@@ -6096,10 +6235,11 @@
           "placement_group[cloud_specific_attributes]": {
             "class": "Hash",
             "non_blank": true,
-            "description": "Cloud specific attributes that have no generic rightscale abstraction"
+            "description": "Cloud specific attributes that have no generic rightscale abstraction",
+            "mandatory": false
           },
           "placement_group[cloud_specific_attributes][account_type]": {
-            "class": "Array",
```

Comment:
Yup. string is correct according to http://reference.rightscale.com/api1.5/resources/ResourcePlacementGroups.html.